src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -12,10 +12,11 @@ import getOriginContext from '../contexts/RequestContext/getOriginContext';
 import getEnv from '../contexts/RequestContext/getEnv';
 import GlobalStyle from '../lib/globalStyles';
 
-const PageWrapper = ({ bbcOrigin, children, id, service, isAmp }) => {
+const PageWrapper = ({ bbcOrigin, children, id, service, isAmp, route }) => {
```

Comment:
I'd suggest just passing the `pageType` value to this wrapper, instead of the whole route object.